bigtable/tests/system.py

```diff
@@ -192,23 +192,55 @@ def test_create_instance(self):
         self.assertEqual(instance.type_, instance_alt.type_)
         self.assertEqual(instance.labels, instance_alt.labels)
 
-    def test_update(self):
+    def test_update_display_name_and_labels(self):
         OLD_DISPLAY_NAME = Config.INSTANCE.display_name
         NEW_DISPLAY_NAME = 'Foo Bar Baz'
+        NEW_LABELS = {'foo_bar': 'foo_bar'}
         Config.INSTANCE.display_name = NEW_DISPLAY_NAME
+        Config.INSTANCE.labels = NEW_LABELS
         Config.INSTANCE.update()
 
         # Create a new instance instance and reload it.
-        instance_alt = Config.CLIENT.instance(INSTANCE_ID)
+        instance_alt = Config.CLIENT.instance(INSTANCE_ID, LABELS)
```

Comment:
Please add 'labels=' before `LABELS`

Reply:
Also, please check that `instance_alt.labels == LABELS` rather than checking `!= NEW_LABELS`

Reply:
Done